config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -205,3 +210,25 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api
       testgrid-tab-name: capi-pr-e2e-k8s-latest-main
+  - name: pull-cluster-api-operator-verify
```

Comment:
Why not using the same approach we are using for CAPD (calling make targets from a script)

https://github.com/kubernetes-sigs/cluster-api/blob/8bc65c3aa70c416b871b02bfe9cd31e17fd10737/scripts/ci-make.sh#L25?

This avoid to add complexity to test grid and it will make things to nicely fit when we are going to merge back on main